README.md

```diff
@@ -8,12 +8,44 @@ CND helps you achieve this with a mix of kubernetes automation, file synching be
 
 ## Quickstart
 
-- Install `cnd` locally 
-- Deploy a kubernetes service
-- Run `cnd up`
-- 
+Let's start with something simple to show you the power of cloud native development. First, install [cnd locally](#installation).  
 
-For a more advanced scenario [check out our voting app cnd demo](https://github.com/okteto/cnd-voting-demo).
+Clone the `welcome` service.
```

Comment:
Should we rename the service to something like `voting`? It's no longer a `welcome web page`

Reply:
Agree, maybe we can call it `voting-app`.
```suggestion
Clone the `voting-app` service.
```